spec/controllers/customers.spec.js

```diff
@@ -0,0 +1,64 @@
+var request = require("request")
+var baseUrl = "http://localhost:3000"
+
+describe("CustomersController", function() {
+  var url = function(endpoint) {
+    return baseUrl + "/customers" + endpoint
+  }
+
+  describe("#getCustomers", function(done) {
+    it("returns a Success response", function(done) {
+      request.get(url("/"), function(error, response, body) {
+        expect(response.statusCode).toBe(200)
+        done()
+      })
+    })
+
+    it("returns JSON", function(done) {
+      request.get(url("/"), function(error, response, body) {
+        expect(response.headers['content-type']).toContain('application/json')
+        done()
+      })
+    })
+
+    it("should be an array of objects", function(done) {
+      request.get(url("/"), function(error, response, body) {
+        var data = JSON.parse(body)
+        expect(typeof data).toEqual('object')
+
+        for (var record of data) {
+          expect(Object.keys(record)).toEqual(['id'])
+        }
+        done()
+      })
+    })
+  })
+
+  describe('#getCustomersSorted', function(done) {
+    it('returns a success response', function(done) {
+      request.get(url("/sort/name?n=10&p=1"), function(error, response, body) {
+        expect(response.statusCode).toBe(200)
+        done()
+      })
+    })
+
+    it("returns JSON", function(done) {
+      request.get(url("/sort/name?n=10&p=1"), function(error, response, body) {
+        expect(response.headers['content-type']).toContain('application/json')
+        done()
+      })
+    })
+
+    it("should be an array of objects", function(done) {
```

Comment:
It may also be worth testing the actual output is what you expect, not just the type you expect.